site/Calendar.php

```diff
@@ -0,0 +1,29 @@
+<html>
+	<head>
+	<link rel="stylesheet" type="text/css" href="Style.css">
+	<link href="https://fonts.googleapis.com/css?family=Raleway"
+	rel="stylesheet">
+		<title>
+			St. Louis Theatre Calendar
+		</title>
+	</head>
+	<body>
+		<div class="Header">
+			<?php include 'include/header.php';
+				PrintHeader();
+			?>
+		</div>
+		<div class="PageTitle">
```

Comment:
Even this div could go in the `PrintHeader` function if you're using the same format on every page.